testing/src/test/java/io/stargate/it/CQLTest.java

```diff
@@ -350,6 +355,46 @@ public void tupleTest()
         assertThat(tupleReturnValue.getInt(2)).isEqualTo(2);
     }
 
+    @Test
+    public void compressionTest()
+    {
+        Arrays.stream(new String[] {"lz4", "snappy"}).forEach(compression -> {
```

Comment:
What do you think about using JUnit 5 parameters? It should be possible to apply them on a per-test method basis.

Reply:
+1
And in this case the parameters are compile-time constant so you can provide them with a simple annotation:
```java
    @ParameterizedTest
    @ValueSource(strings = { "lz4", "snappy" })
    public void compressionTest(String compression) { ... }
```

Reply:
Makes sense, will do.